src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificate/HealthCertificateViewModel.swift

```diff
@@ -143,14 +143,31 @@ final class HealthCertificateViewModel {
 			title = AppStrings.HealthCertificate.Details.euCovidCertificate
 			subtitle = AppStrings.HealthCertificate.Details.RecoveryCertificate.title
 		}
+		let attributedTitle = NSAttributedString(
+			string: title,
+			attributes: [
+				.font: UIFont.enaFont(for: .body),
+				.foregroundColor: UIColor.enaColor(for: .textContrast),
```

Comment:
nit pik:

`UIColor.` implicite

Reply:
it wouldn't compile, i get a compiler error "Reference to member 'body' cannot be resolved without a contextual type"

Reply:
it wouldn't compile, i get a compiler error "Reference to member 'body' cannot be resolved without a contextual type" if i use  .body or .textContrast

Reply:
@30mar you will write .enaColor(for: .textContrast) or .enaFont(for: .body), it should work.